packages/react-ai/src/components/AIConversation/views/Controls/FormControl.tsx

```diff
@@ -272,6 +283,9 @@ export const FormControl: FormControl = () => {
         isLoading={isLoading}
         error={error}
         setError={setError}
+        onKeyDown={handleOnKeyDown}
+        onCompositionStart={() => setComposing(true)}
+        onCompositionEnd={handleOnCompositionEnd}
```

Comment:
Why are these event callbacks typed for a `textarea` but being added to a `Form` interface?

Reply:
This part was built on Danny's approach with the AI docs PR [here](https://github.com/aws-amplify/amplify-ui/blob/f5a6a16c10d735df7dddf8d639877e9a57c185d0/packages/react-ai/src/components/AIConversation/views/Controls/FormControl.tsx#L344). Should we discuss with Danny on moving the handle to a different file?